Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml

```diff
@@ -36,3 +36,4 @@
   - type: SpentAmmoVisuals
     state: emp
     suffix: false
+
```

Comment:
What?

Reply:
that was because, something mess up and it added things that wasnt post to be added even tho i took it off. So I revert it back to the OG file